cmd/wallet-sdk-gomobile/api/vcparse/vcparse.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I wanted to have this function in the same place as the VC definition, but this wasn't possible due to a circular import issue. I can think of a way around it, but it would result in a lot of code duplication.